src/Form/Formbot.js

```diff
@@ -110,15 +114,27 @@ export default class Formbot extends React.Component {
 
   validateField(field) {
     return new Promise(resolve => {
-      const validation = this.props.validations[field];
+      const fieldState = this.state.fields[field] || {};
```

Comment:
Could we use lodash `get` to simplify this?

Reply:
it looks like we're not using lodash at all so i dont want to add it just for this instance